guide/plg_atTheTelescope.tex

```diff
@@ -933,7 +933,10 @@ \subsection{ASCOM (Windows only)}
 and most of the astronomy software and tools can communicate with
 ASCOM compatible devices.
 
-The ASCOM telescope support has been added to Stellarium's Telescope Control plugin by Gion Kunz in version 0.19.3.
+The ASCOM telescope support has been added to Stellarium's Telescope Control plugin by Gion Kunz in version 0.19.3.\footnote{%
+Unfortunately ASCOM seems to be incompatible with Qt6. 
```

Comment:
Not ASCOM, but "COM ASCOM" or "ASCOM COM". This is the name their official website uses for this Windows-specific interface (the other being Alpaca).

Reply:
OK. 20 years ago ASCOM was presented, the COM part of the name coming from Windows-specific COM. Maybe "traditional ASCOM". So far we do not mention Alpaca.